java/client/src/main/java/glide/api/commands/GenericCommands.java

```diff
@@ -54,4 +54,22 @@ public interface GenericCommands {
      * }</pre>
      */
     CompletableFuture<Object[]> exec(Transaction transaction);
+
+    /**
+     * Move <code>key</code> from the currently selected database to the database specified by <code>
+     * dbIndex</code>.
+     *
+     * @see <a href="https://redis.io/commands/move/">redis.io</a> for more details.
+     * @param key The key to move.
+     * @param dbIndex The index of the database to move <code>key</code> to.
+     * @return <code>true</code> if <code>key</code> was moved, or <code>false</code> if the <code>key
+     *     </code> already exists in the destination database or does not exist in the source
+     *     database.
```

Comment:
Can you remove this line break?
```suggestion
     *     </code> already exists in the destination database or does not exist in the source database.
```

Reply:
no, spotless enforces the line break